hack/make-rules/test-integration.sh

```diff
@@ -1,3 +1,4 @@
+es
```

Comment:
??

Reply:
Don't ask... no idea where that was supposed to go. Certainly not into the commit.